### PR TITLE
Added support for implicit FTPS and gem dependencies

### PR DESCRIPTION
**Pull request by swibowo**
#### General

- [ ] Update Changelog 

- [ ] RuboCop passes

- [ ] Existing tests pass 

#### Purpose

Added support for implicit FTPS. Also added gem dependencies.

#### Known Compatablity Issues

None.

### CODE REVIEW

File: bin/check-ftp.rb [hunk 1]
```diff
@@ -63,6 +63,10 @@ class CheckFTP < Sensu::Plugin::Check::CLI
          short: '-t SECS',
          proc: proc(&:to_i),
          default: 15
+  option :implicit,
```
Comment (eheydrick) [inline bin/check-ftp.rb]: I think `implicit_ftps` would be a better name for this option. It's more clear.

File: sensu-plugins-ftp.gemspec [hunk 2]
```diff
@@ -29,7 +29,9 @@ Gem::Specification.new do |s|
   s.test_files             = s.files.grep(%r{^(test|spec|features)/})
   s.version                = SensuPluginsFtp::Version::VER_STRING
 
-  s.add_runtime_dependency 'sensu-plugin', '~> 1.2'
+  s.add_runtime_dependency 'sensu-plugin',                  '~> 1.2'
+  s.add_runtime_dependency 'net',                           '~> 0.3.3'
```
Comment (eheydrick) [inline sensu-plugins-ftp.gemspec]: The net gem "Retrieves information for Twitter users". Did you mean something else?

**Review by eheydrick — CHANGES_REQUESTED** [review]

File: bin/check-ftp.rb (hunk 1 above)
Comment (swibowo) [reply bin/check-ftp.rb]: Good point. Changed the option name.

File: sensu-plugins-ftp.gemspec (hunk 2 above)
Comment (swibowo) [reply sensu-plugins-ftp.gemspec]: Oh man, I added it by accident because I have the gem installed on my env. Removed.

**Comment by swibowo** [comment]
@eheydrick should be good now. Thanks for reviewing.

**Review by eheydrick — APPROVED** [review]

**Comment by eheydrick** [comment]
Thanks for the contribution @swibowo! It's released in 1.1.0, now live on rubygems.

**Pull request merged**